Flutter/calendar/appointments.md

```diff
@@ -130,6 +130,132 @@ events.notifyListeners(CalendarDataSourceAction.reset, null);
 {% endhighlight %}
 {% endtabs %}
 
+## DataSource manipulation
+
+By call this method whenever the datasource changes, to notify the datasource may have. Need to notify the datasource action(add, remove and reset) through notify listener.
+
+#### Add Action
+
+If the appointment is added in an underlying datasource, you can call the notifyListeners.
+
+In the following example, the appointment is added and notifyListeners is called in onPressed callback of the FloatingActionButton. 
+So, users can call notifyListeners methods to refresh the dataSource.
```

Comment:
In the following example, the appointment is added and notifyListeners is called in onPressed callback of the FloatingActionButton. 
So, users can call notifyListeners methods to refresh the dataSource.


notifyListeners. explained twice, optimize and improve the content

Reply:
I have checked the content related to notifyListener. I have optimized the content. I have updated the changes in latest commit.